deploy/helm/values/dev.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for cfe-crime dev
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+resources:
+  limits:
+    cpu: 900m
+    memory: 1G
+  requests:
+    cpu: 500m
+    memory: 900Mi
```

Comment:
This reserves 1GB of ram on the cluster for a pod, which seems excessive.
In Grafana, I'm seeing cfe-crime pods in a steady state using between 100 and 121GB, and start-up peak above that. So I suggest you set requests.memory at 150GB.

The memory limit is fine.

<img width="1562" alt="Screenshot 2023-06-07 at 15 14 23" src="https://github.com/ministryofjustice/cfe-crime/assets/307612/f10d5451-0819-4ddb-95c0-5bfbefbe55ab">

Reserving half a CPU for a pod might be a about right. The graph tells us they idle around 50-80m, but when they kick into action, if start-up is anything to go by, they happily devour 3 CPUs, so a limit of 0.9 CPUs may be fine - they will be perhaps constrained, but we can increase it if it seems slow.
<img width="1558" alt="Screenshot 2023-06-07 at 15 23 59" src="https://github.com/ministryofjustice/cfe-crime/assets/307612/c2a6f770-550a-479d-aa77-0b4887c583a0">